Presentation.Web/app/services/exportGridToExcelService.ts

```diff
@@ -148,6 +129,36 @@
             );
         }
 
+        private mapShownColumns(columns: IKendoGridColumn<any>[]) {
+            columns.filter(x => !x.hidden).forEach(column => {
+                this.columnsToShow.push({ columnId: column.persistId });
+            }, this);
+        }
+
+        private moveRelatedColumns(columns: IKendoGridColumn<any>[], e: IKendoGridExcelExportEvent<any>) {
+            columns.filter(x => x.hidden).forEach(column => {
+                if (column.parentId === undefined)
+                    return;
+                //look for a parent column and check if any was found, and if any parent column should be visible
+                var columnsWithMatchingParentId = columns.filter(x => x.persistId === column.parentId);
+                if (columnsWithMatchingParentId.length !== 1) {
+                    console.error("Column has multiple columns with matching parentId");
```

Comment:
or none.. `!== 1`  :-). Perhaps just change the error to reflect what actually is the case.. no parent found or mulitple parents found